test/pouch_api_create_test.go

```diff
@@ -0,0 +1,51 @@
+package main
+
+import (
+	"encoding/json"
+	"net/url"
+
+	"github.com/alibaba/pouch/apis/types"
+	"github.com/alibaba/pouch/test/environment"
+	"github.com/alibaba/pouch/test/request"
+
+	"github.com/go-check/check"
+)
+
+// PouchAPICreateSuite is the test suite for info related API.
+type PouchAPICreateSuite struct{}
+
+func init() {
+	check.Suite(&PouchAPICreateSuite{})
+}
+
+// SetUpTest does common setup in the beginning of each test.
+func (suite *PouchAPICreateSuite) SetUpTest(c *check.C) {
+	SkipIfFalse(c, environment.IsLinux)
+}
+
+// TestCreateOk test create api is ok with default parameters.
+func (suite *PouchAPICreateSuite) TestCreateOk(c *check.C) {
+
+	// must required
+	q := url.Values{}
+	q.Add("name", "api-create-test")
+
+	cc := types.ContainerConfig{
+		Image: busyboxImage,
+	}
+	hc := types.HostConfig{}
+	obj := types.ContainerConfigWrapper{
+		ContainerConfig: cc,
+		HostConfig:      &hc,
+	}
```

Comment:
I do not think it is a good way to utilize package /apis/types. Here is my reason: package types is used to transfer data from server side and client side. It is a contract between them. While when we test API servers, we should use raw data to construct a body, not from defined types, but raw constructed types, like: `map[string]interface{}{}`.